app/helpers/catalog_admin/catalogs_helper.rb

```diff
@@ -38,6 +38,10 @@ def catalog_access_label(catalog)
     [:everyone, :members, :catalog_staff][catalog_access(catalog) - 1].to_s
   end
 
+  def catalog_access_open_for_everyone_select_id
```

Comment:
Maybe a constant is a better fit for this type of data 🙂

Reply:
maybe maybe maybe

c'était chiant: https://github.com/catima/catima/pull/583/commits/76ef52fc164af0129b9a14548b76d8a9acde3d67